packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -651,6 +651,13 @@ export class DeltaManager
                     this.client.mode = requestedMode;
                     connection = await docService.connectToDeltaStream(this.client);
                 } catch (origError) {
+                    if (typeof origError === "object" && origError !== null &&
```

Comment:
It would be great to have a telemetry event here (or maybe in the driver) to record the reason why we are using this mode